src/pages/ArticlePage.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState} from "react";
+import { useParams } from "react-router-dom";
+import axios from "axios";
+import './ArticlePage.css';
+
+const ArticlePage = () => {
+    const { article_id } = useParams();
+    const [article, setArticle] = useState(null);
+    const [loading, setLoading] = useState(true);
+    const [error, setError] = useState(null);
+
+    useEffect(() => {
+        axios.get(`https://jesjin-nc-news.onrender.com/api/articles/${article_id}`)
```

Comment:
You can extract this out into it's own function to neaten up the page a bit! Check out the axios notes